analyses/interaction-plots/scripts/03-plot_interactions.R

```diff
@@ -74,12 +90,22 @@ cooccur_df <-
 
 labels <- unique(c(cooccur_df$label1, cooccur_df$label2))
 
+# check the order of the labels to be decreasing by mut count 
```

Comment:
The string manipulation through here is because you have `<gene symbol> (<count>)` correct?

Reply:
That is correct. It is a bit of a hack, if I'm being honest, but it prevented having to rewrite previous code.